core/src/test/scala/org/broadinstitute/dsde/rawls/webservice/SubmissionApiServiceSpec.scala

```diff
@@ -85,7 +86,11 @@ class SubmissionApiServiceSpec extends ApiServiceSpec {
     try {
       testCode(actor)
     } finally {
+      // stops actor and waits for it to shut down
```

Comment:
Seems to work -- I lifted it from akka docs:
http://doc.akka.io/docs/akka/snapshot/scala/testing.html#watching-other-actors-from-probes